doc/sphinx/src/building.rst

```diff
@@ -173,7 +172,7 @@ The below example ``CMakeLists.txt`` can be used to compile the
    cmake_minimum_required(VERSION 3.11)
 
    project(parthenon_linking_example)
-   set(Kokkos_CXX_STANDARD "c++14")
+   set(CMAKE_CXX_STANDARD "17")
```

Comment:
I don't think quotation marks are needed here, but not 100% sure.

Reply:
Yes, I'll fix.